country_config/null_download.json

```diff
@@ -5,34 +5,36 @@
 
   "rawIntro":"Use this form to download raw data for use in your own analysis. Select which form's data you would like to download, and optionally further select which attributes you would like to download. ",
 
-	"prepared":[{
-		"title":"Consultations",
-		"api":"/api/export/data_table/test/reg_1",
-		"access":"cd",
-		"variables":[
-			["reg_1","consultations"]
-		],
-		"group_by": [
-			["epi_year", "Year"],
-			["clinic:location", "Clinic"],
-			["epi_week", "Week"]
-		]
-		
-
-	},{
-      "title":"Communicable Diseases",
-      "api":"/api/export/category/demo_case/cd_tab/communicable_diseases",
-      "name_category": "individual_icd",
-      "access":"cd",
-      "fields":[
-          ["intro./visit","Visit Type"],
-          ["pt./visit_date$date","Visit Date"],
-          ["pt./visit_date$year","Year"],
-          ["pt./visit_date$month","Month"],
-          ["pt./visit_date$epi_week","Epi Week"],
-          ["region","Region"],
-          ["district","District"],
-          ["clinic","Clinic"],
+    "prepared":[
+        {
+	    "title":"Completeness",
+	    "api":"/api/export/week_level/test/clinic",
+	    "access":"cd",
+            "wide_data_format":1,
+	    "variable":
+                ["completeness:/completeness/reg_1/1/4/<start_week>/5,6/reg_1/<end_date>", "Completeness"]
```

Comment:
This is a two column download, what is in each column?
...or wait, what is this new variable field? And wide data format?

Reply:
So this is a completely new type of download data. We want to download data by clinic, week and year. The variable column has the actual variable and then the display name. As we have done in all the download data variables. 

data_format can be long or wide. This is a flag to say we want it wide where we have multiple observations in the same row. 

Long: 
clinic, week, value
1, 1, 23
1, 2, 56

Wide: 
        
clinic, Week 1, Week 2, ....
1,         23,        56